README.md

```diff
@@ -71,9 +71,9 @@ some excerpt
 
 + **layout**: Set to ***post***.
 + **category**: Choose one of the following categories: ``Getting Started``, ``Ingesting Data``, ``Processing Data``, ``Data Output``, ``Demos``, ``Connectors``.
-+ **tags**: Put as many as are relevant. Tags will be shown on the Tags page.
++ **tags**: Put as many as are relevant. Tags will be shown on the Tags page. All tags must be lowercase.
 + **subtitle**: This will show beneath the title on the post details page.
-+ **technologies**: List what technologies(less or equal than 2) is used in this post. This will show on the right panel.
++ **technologies**: List what technologies(less or equal than 2) is used in this post. This will show on the right panel. The first letter of each technology should be capitalized. 
```

Comment:
can we reword this to say "Technologies listed here should be appropriately capitalized." -- we don't want to force just the first letter; i.e. if it's an acronym, like SDP, or something like eTag

Reply:
That makes sense. I just make the changes to the README.